### PR TITLE
Add initial site

### PR DESCRIPTION
**Pull request by alexandesigner**
Initial site of the blaze in this PR contains a prototype website (we still need a landing page).. but it's a start ... explore some of the material in the documentation and the guide using theme base ``hexo-theme-meteor`` the official guide Meteor;

### CODE REVIEW

File: site/source/guide.md [hunk 1]
```diff
@@ -0,0 +1,672 @@
+---
```
Comment (mitar) [inline site/source/guide.md]: This is copied from Blaze guide as it is?

File: site/source/index.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
+---
```
Comment (mitar) [inline site/source/index.md]: This is new or copied from somewhere?

File: site/source/guide.md (hunk 1 above)
Comment (alexandesigner) [inline site/source/guide.md]: Exactly @mitar

File: site/source/index.md (hunk 2 above)
Comment (alexandesigner) [inline site/source/index.md]: This is basically the Readme in the package in Atmosphere ... that is what we are using in the readme too .. 

_I think we could add content to complete talking a little bit about the Blaze API ._ 😄 😄

Comment (mitar) [inline site/source/index.md]: Hm, so we will have `meteor/blaze/README.md` and `meteor/blaze/packages/blaze/README.md` (which will go to Atmosphere) and then this file. This feels to me like ...  a bit of duplication. How can we make sure that those all files are reasonable and in sync somehow?

**Comment by stubailo** [comment]
After this is published, should we redirect the meteor guide page about blaze to this site?

**Comment by mitar** [comment]
Yes, we could do that. We should probably then remove it from repository there as well. I would really worry to have duplicate efforts at two places.

What about Meteor documentation? And API generation? We should figure that out as well? Like how to make "link to source code" work?

**Comment by stubailo** [comment]
Yeah I agree we should not duplicate content!

Let me know when we can help with that. I wrote the original JSDoc stuff but @tmeasday implemented the current build process for the API docs.

**Comment by mitar** [comment]
OK, so let's start first to have automatic building and deploying. And then just keep in mind to not allow changes to Blaze related content to guide and docs but direct them here. And then we can fix links and other stuff and then link it from Meteor content.

**Comment by mitar** [comment]
I will merge this for now, so that we get a version up. And then we can work on de-duplicating content. Maybe simply Atmosphere readme should be short and point to blazejs.org (because in the future it will be just one type of a package for a general Blaze package).

**Comment by mitar** [comment]
BTW, I see that you have `site/source/logo/main-logo.png`, do we have to duplicate this image? Should we move also png files to `images/` and then source it from there for documentation as well?

**Comment by tmeasday** [comment]
For API docs, I would suggest we move these two files
 - https://github.com/meteor/docs/blob/master/source/api/blaze.md
 - https://github.com/meteor/docs/blob/master/source/api/templates.md

Into your hexo theme, and put a pointer in the Meteor docs. You'll need to setup the hexo theme with the api-box helper and the code as a submodule.

I can do that last part if you'd prefer, just let me know where to put it.

**Comment by tmeasday** [comment]
PS if you'd like to use our S3 deployment script for hexo sites, we can host the bucket in our S3 account.

**Comment by mitar** [comment]
> PS if you'd like to use our S3 deployment script for hexo sites, we can host the bucket in our S3 account.

I was thinking of using GitHub pages.

**Comment by tmeasday** [comment]
Sure, whatever wroks

**Comment by mitar** [comment]
> Into your hexo theme, and put a pointer in the Meteor docs. You'll need to setup the hexo theme with the api-box helper and the code as a submodule.
> 
> I can do that last part if you'd prefer, just let me know where to put it.

I think that would be great if you could help. There is already an empty section for API. If you could add it there it would be great. As for files themselves, I think that for now we could add them along with other files in the docs (so `/site` directory).

**Comment by mitar** [comment]
Deployed: http://blazejs.org/

**Pull request merged**